e2e/orchestrator_filter_path_nested_test.go

```diff
@@ -249,18 +250,17 @@ func testOrchestratorFilterPathNested(executionStream pb.Execution_StreamClient,
 				})
 				require.NoError(t, err)
 			})
-			t.Run("wait 5 sec to check execution is not created", func(t *testing.T) {
-				ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(5*time.Second))
-				defer cancel()
-				executionStream, err := client.ExecutionClient.Stream(ctx, &pb.StreamExecutionRequest{})
-				require.NoError(t, err)
-				_, err = executionStream.Recv()
-				require.Contains(t, err.Error(), context.DeadlineExceeded.Error())
+			t.Run("wait timeout to check execution is not created", func(t *testing.T) {
+				require.PanicsWithError(t, fmt.Sprintf("pollExecutionOfProcess timeout with process hash %q and status %q and nodeKey %q", processHash, execution.Status_InProgress, "n2"), func() {
+					pollExecutionOfProcess(processHash, execution.Status_InProgress, "n2")
```

Comment:
I'm not really sure about that but this is a blocking function isn't it? so why having to panic where you could just return an error?

Reply:
fixed in https://github.com/mesg-foundation/engine/pull/1758